binder/json.py

```diff
@@ -48,14 +48,12 @@ def jsondumps(o, default=default, indent=None):
 	return json.dumps(o, default=default, indent=indent)
 
 
-
 def jsonloads(data):
 	try:
-		return json.loads(data.decode())
+		return json.loads(data)
```

Comment:
jsonloads is also used a lot on standalone. Will this potentially break all that usage?

Reply:
As addressed in the PR description: depends on the python version.
Since python 3.6 `json.loads` accepts both `str` and `bytes` so the `decode` is bultin when its `bytes`. With earlier versions that would indeed break, but according to the setup.py those are not supported anyway.